doc/user/ospfd.rst

```diff
@@ -510,12 +512,14 @@ Interfaces
 
 
    Enable OSPF on the interface, optionally restricted to just the IP address
-   given by `ADDR`, putting it in the `AREA` area. Per interface area settings
-   take precedence to network commands
-   (:clicmd:`network A.B.C.D/M area A.B.C.D`).
-
-   If you have a lot of interfaces, and/or a lot of subnets, then enabling OSPF
-   via this command may result in a slight performance improvement.
+   given by `ADDR`, putting it in the `AREA` area. If you have a lot of
```

Comment:
We should use double ticks to highlight the word.

Reply:
I double checked the use of ticks and double ticks in the file. We use double ticks to highlight code or commands, while a single tick is used for keywords like we have here.

Reply:
rendered examples: http://docs.frrouting.org/en/stable-8.1/ospfd.html#clicmd-ip-ospf-area-AREA-ADDR